megamek/src/megamek/common/battlevalue/JumpShipBVCalculator.java

```diff
@@ -779,96 +457,30 @@ public static int calculateBV(Jumpship jumpShip, boolean ignoreC3, boolean ignor
         }
         double speedFactor = Math.pow(1 + (((double) runMp - 5) / 10), 1.2);
         speedFactor = Math.round(speedFactor * 100) / 100.0;
+        bvReport.addLine("Final Speed Factor:", "", speedFactor);
 
-        bvText.append(startRow);
-        bvText.append(startColumn);
-
-        bvText.append("Final Speed Factor: ");
-        bvText.append(endColumn);
-        bvText.append(startColumn);
-        bvText.append(endColumn);
-        bvText.append(startColumn);
-        bvText.append(speedFactor);
-        bvText.append(endColumn);
-        bvText.append(endRow);
-
+        double obv; // offensive bv
         obv = weaponBV * speedFactor;
-
-        bvText.append(startRow);
-        bvText.append(startColumn);
-
-        bvText.append("Weapons BV * Speed Factor ");
-        bvText.append(endColumn);
-        bvText.append(startColumn);
-
-        bvText.append(weaponBV);
-        bvText.append(" * ");
-        bvText.append(speedFactor);
-        bvText.append(endColumn);
-        bvText.append(startColumn);
-        bvText.append(" = ");
-        bvText.append(obv);
-        bvText.append(endColumn);
-        bvText.append(endRow);
-
-        bvText.append(startRow);
-        bvText.append(startColumn);
+        bvReport.addLine("Weapons BV * Speed Factor ",weaponBV + " * " + speedFactor, "", obv);
 
         double finalBV;
         if (jumpShip.useGeometricMeanBV()) {
-            bvText.append("2 * sqrt(Offensive BV * Defensive BV");
             finalBV = 2 * Math.sqrt(obv * dbv);
             if (finalBV == 0) {
                 finalBV = dbv + obv;
             }
-            bvText.append("2 * sqrt(");
-            bvText.append(obv);
-            bvText.append(" + ");
-            bvText.append(dbv);
-            bvText.append(")");
+            bvReport.addLine("2 * sqrt(Offensive BV * Defensive BV","2 * sqrt(" + obv + " + " + dbv + ")", "");
         } else {
-            bvText.append("Offensive BV + Defensive BV");
             finalBV = dbv + obv;
-            bvText.append(obv);
-            bvText.append(" + ");
-            bvText.append(dbv);
+            bvReport.addLine("Offensive BV + Defensive BV",obv + " + " + dbv, "");
```

Comment:
```suggestion
            bvReport.addLine("Offensive BV + Defensive BV", obv + " + " + dbv, "");
```

Reply:
The IntelliJ parameter helpers hide these missing spaces